critiquebrainz/frontend/templates/bb_literary_work/entity.html

```diff
@@ -60,6 +60,52 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
         {% endif %}
       </ul>
     {% endif %}
+
+    {% if work_rels_info is defined and work_rels_info %}
+      <h4>{{ _('Translations') }}</h4>
+      <table class="table table-condensed table-hover">
+        <thead>
+          <tr>
+            <th>{{ _('Name') }}</th>
+            <th>{{ _('Languages') }}</th>
+            <th>{{ _('Type') }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for work in work_rels_info %}
```

Comment:
it would be nice to give this variable a clearer name to say that it's translation relationships